generator/internal/genclient/translator/protobuf/testdata/scalar.proto

```diff
@@ -0,0 +1,50 @@
+syntax = "proto3";
```

Comment:
nit: almost certainly this will need some copyright boilerplate.  All files that can have the copyright boilerplate are supposed to have it.

Reply:
ack